src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt

```diff
@@ -549,7 +549,6 @@ class PrisonerIepLevelReviewServiceTest {
   private val iepTime: LocalDateTime = LocalDateTime.now().minusDays(10)
   private fun iepSummary(iepLevel: String = "Enhanced", iepDetails: List<IepDetail> = emptyList()) = IepSummary(
     bookingId = 1L,
-    daysSinceReview = 60,
```

Comment:
`daysSinceReview` **is** in fact provided by prison api. it can't be here as is because this mock already presumes that prison api's JSON has been deserialised.

question: does the deserialisation system ignore superfluous/unexpected fields?

Reply:
seems the answer is that the superfluous/unexpected field _is_ ignored as proved by tests using this stub still passing:
https://github.com/ministryofjustice/hmpps-incentives-api/blob/f8e4563969640d393ccc5e81e0dce3715c9afe60/src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/wiremock/PrisonApiMockServer.kt#L141